lib/src/timetable.dart

```diff
@@ -16,6 +16,13 @@ typedef AllDayEventBuilder<E extends Event> = Widget Function(
   AllDayEventLayoutInfo info,
 );
 
+/// Signature for [Timetable.weekIndicatorBuilder] and
+/// [Timetable.weekIndicatorBuilder] params
+typedef HeaderBuilder = Widget Function(
```

Comment:
Could you please rename this to `HeaderWidgetBuilder`?

Reply:
done